.env.example

```
@@ -124,28 +122,15 @@ FIRECRAWLER_HOST=localhost
 FIRECRAWLER_PORT=8000
 FIRECRAWLER_TRANSPORT=streamable-http
 FIRECRAWL_API_URL=http://localhost:3002
+MCP_PORT=5100
+MCP_FIRECRAWL_API_URL=http://firecrawl-api:3002
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_FIRECRAWL_API_URL key should go before the MCP_PORT key

(UnorderedKey)